src/js/tabs/motors.js

```diff
@@ -126,14 +126,14 @@ TABS.motors.initialize = function (callback) {
             }
         }
         while (data[0].length > 300) {
-            for (i = 0; i < data.length; i++) {
-                data[i].shift();
+            for (let i = 0; i < data.length; i++) {
+                index.shift();
```

Comment:
Where's this index from?

Please at least check your code for errors before pushing.

Reply:
Oops this was the only was I didn't test after changes. Sorry. GUI was working, didn't look at the console :-( 
Fixed.